pkgs/top-level/all-packages.nix

```diff
@@ -5127,7 +5127,9 @@ in
 
   rgbds = callPackage ../development/compilers/rgbds { };
 
-  rtags = callPackage ../development/tools/rtags/default.nix {};
+  rtags = callPackage ../development/tools/rtags/default.nix {
+    inherit (darwin) apple_sdk;
```

Comment:
The attributes exist on every platform, they don't evaluate but that's fine if they are not used.